documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-installing-certs-per-listener-{context}']
+= Installing your own CA certificates for listeners
+
+You can install your own certificates and public keys for Kafka brokers on a per-listener basis. These certificates can be signed by an external Certificate Authority (CA), such as your company's internal CA or a public CA, rather than the cluster or clients CAs (which are internal to {ProductName}). Installing per-listener certificates is supported for two types of listeners:
+
+* TLS listeners for inter-cluster communication
+
+* External listeners (including `route`, `loadbalancer`, and `nodeport` types) for communication between Kafka clients and Kafka brokers
+
+Installing your own certificates and keys for plain listeners is not supported.
+
+.Prerequisites
+
+* A Kubernetes cluster.
+* The Cluster Operator is running.
+* Public key of the external CA.  
+* Your own X.509 certificate in CRT or PEM format.
```

Comment:
CRT is not a format. It's just a possible extension for a certificate formatted with PEM. Just fyi, other formats are DER, PKCS12, etc etc. In this guess it's enough saying "...in PEM format".